Gopkg.lock

```diff
@@ -812,6 +812,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "284ad58436782380c1524b9794fbca2cc798fbdeab89818fdd22a063398d9e19"
+  inputs-digest = "843238741f4e88d78a8acaa703af920f93880a14b119627c75831c56a6e20393"
```

Comment:
Why this hash auto-updated? There is no deps change in Gopkg.toml, right?

Reply:
I did a `make deps`? because I switched branches.

Reply:
ok